npm/design-system/README.md

```diff
@@ -0,0 +1,77 @@
+# @cypress/design-system
+
+A design system for the surfaces of testing softwares. 🐛💅
+
+## Values
+
+### It is discreet
+
+Let the surfaces fade away to allow work to happen.
+
+### It is native
+
+Let the surfaces be familiar to where work happens.
+
+### It is accessible
+
+Let the surfaces honor accepted standards so everyone can use them.
+
+### It is specialized
+
+Let the surfaces be appropriate for the job to be done. Favor consistency over novelty, but not at the cost of functionality.
+
+## Usage
+The components work with or without the global stylesheet import. The stylesheet import is used to setup global scss tokens, colors, utility classes, and typography.
+
+Component Usage:
+
+```jsx
+import { Button } from '@cypress/design-system'
+```
+
+To setup global CSS tokens and mixins, you can import the library's `index.scss`. You can either do this with `@use` or `@import` (See CSS Trick's intro to Sass modules [here](https://css-tricks.com/introducing-sass-modules/#import-files-with-use))
+
+SCSS usage:
+
+```scss
+// scoped within the *.scss file
+@use '@cypress/design-system/src/index.scss' as *;
+
+// import variables and mixins throughout the whole project
+// or @import('@cypress/design-system/src/index.scss');
+
+.my-component {
+  text-color: $accent-color-01;
+}
+```
+
+## Development
+We are currently using:
+* CSS Modules for styling
```

Comment:
Sure this is not the best thing we can use for design system styling. Is this only for testing?

Reply:
What would you prefer? SASS + CSS Modules is my preferred environment, _particularly_ for design systems. Cascading styles or similar using JS is very non-performant.

Reply:
I've been a core maintainer of material-ui for 3 years and partially working on it. And css is really bad for the components library because of different reasons. 

I already described this in slack, you can follow the discussion there